transport/http/inbound.go

```diff
@@ -69,7 +69,7 @@ func Mux(pattern string, mux *http.ServeMux) InboundOption {
 // YARPC.
```

Comment:
Please update the method described to point out the chain of interceptors and they will be called in the same order passed in the options.

Reply:
+1